src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -17,6 +17,10 @@ import type { FiberRoot } from 'ReactFiberRoot';
 import type { PriorityLevel } from 'ReactPriorityLevel';
 import type { ReactNodeList } from 'ReactTypes';
 
+var { isValidElement } = require('ReactElement');
+var invariant = require('invariant');
+var ReactFeatureFlags = require('ReactFeatureFlags');
+
```

Comment:
Actually, not all our tooling works with ES6 imports yet so we can't. require() works fine for us for now.